app/presenters/history_event.rb

```diff
@@ -0,0 +1,47 @@
+require "delegate"
+
+# HistoryEvent wraps & delegates to C2Version
```

Comment:
I added this because there's no other indication that it relates to C2Version, which makes reading it much easier. Or is deducing that from HistoryList enough?